### PR TITLE
Added script which allows to upload a number of files into repository

### PR DESCRIPTION
**Pull request by Korulag**

### CODE REVIEW

File: scripts/upload_to_repository.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                 pprint.pformat(rpm_package_hrefs, indent=4))
+
+    await pulp_client.modify_repository(
+        repository['pulp_href'], rpm_package_hrefs)
+    if not repository['autopublish']:
+        await pulp_client.create_rpm_publication(repository['pulp_href'])
+
+
+def main():
+    arguments = parse_args(sys.argv[1:])
```
Comment (anfimovdm) [inline scripts/upload_to_repository.py:85]: I think that `parse_args` would be enough here

File: alws/utils/file_utils.py [hunk 2]
```diff
@@ -25,9 +25,89 @@ async def download_file(url: str, dest: BinaryIO):
             dest.write(content)
 
 
+async def download_big_file(url: str, dest: BinaryIO, chunk_size: int = 1048576):
```
Comment (anfimovdm) [inline alws/utils/file_utils.py:28]: please left a comment for size in MB

File: alws/utils/file_utils.py [hunk 3]
```diff
@@ -25,9 +25,89 @@ async def download_file(url: str, dest: BinaryIO):
...
+
+    Returns
+    -------
+    _hashlib.HASH
+        Hashlib hashing function.
+    """
+    return hashlib.new('sha1' if checksum_type == 'sha' else checksum_type)
+
+
+def hash_file(file_path, hasher=None, hash_type=None, buff_size=1048576):
```
Comment (anfimovdm) [inline alws/utils/file_utils.py:77]: same

File: alws/utils/pulp_client.py [hunk 4]
```diff
@@ -203,6 +205,42 @@ async def upload_comps(self, data: dict) -> typing.List[str]:
         task_result = await self.wait_for_task(task['task'])
         return task_result['created_resources']
 
+    async def upload_local_file(self, file_path: str, sha256: str,
+                                chunk_size: int = 52428800):
```
Comment (anfimovdm) [inline alws/utils/pulp_client.py:209]: same

**Review by anfimovdm — CHANGES_REQUESTED** [review]

File: alws/utils/pulp_client.py [hunk 5]
```diff
@@ -203,6 +205,42 @@ async def upload_comps(self, data: dict) -> typing.List[str]:
...
+        file_size = os.path.getsize(file_path)
+        chunks = math.ceil(file_size / chunk_size)
+        start = 0
+        upload_href = (await self.request(
+            'POST', 'pulp/api/v3/uploads/', json={'size': file_size}
+        ))['pulp_href']
```
Comment (anfimovdm) [inline alws/utils/pulp_client.py:215]: can we reuse already existent endpoints for `uploads` from `pulp_client`?

**Review by anfimovdm — CHANGES_REQUESTED** [review]

File: alws/utils/pulp_client.py (hunk 5 above)
Comment (Korulag) [reply alws/utils/pulp_client.py:215]: I tried to do as less changes to existing functionality as possible, didn't want to break the `comps` upload. Can rework both of them to use the same code

**Review by anfimovdm — APPROVED** [review]

**Review by javihernandez — APPROVED** [review]

**Pull request merged**